code/modules/xenobio2/mob/slime/slime.dm

```diff
@@ -0,0 +1,83 @@
+/*
+Slime definitions, Life and New live here.
+*/
+/mob/living/simple_animal/xeno/slime //Adult values are found here
+	nameVar = "grey"		//When mutated, nameVar might change.
+	desc = "A shifting, mass of goo."
+	speak_emote = list("garbles", "chirps", "blurbles")
+	colored = 1
+	color = "#CACACA"
+	icon = 'icons/mob/slime2.dmi'
+	icon_state = "slime adult"
+	icon_living = "slime adult"
+	icon_dead = "slime adult dead"
+	internal_vol = 200
+	mut_max = 200
+	mutable = COLORMUT
+	var/is_child = 1
+	var/cores = 3
+	var/growthcounter = 0
+	var/growthpoint = 150 //At what point they grow up.
+	var/shiny = 0
+	move_to_delay = 17 //Slimes shouldn't be able to go faster than humans.
+	default_chems = list("slimejelly" = 5)
+	
+	var/emote_on = null
+	
+	maleable = 2
+	
+	//Slimes can speak all of the languages, oh no!
+	universal_speak = 1
+	speak_chance = 10
+	speak = list( 
+				"Northern Star?",
```

Comment:
```station_name()``` is a proc, rather than a define which is why you cannot use it when initializing objects (due to being non-constant).
It can be added separately in /New() or similar location tho.